### ISSUE TITLE
Hosts without owner forces katprep_snapshot to crash

### ISSUE THREAD

**Issue by stdevel** [stdevel/katprep#82]
Hosts without owners forces ``katprep_snapshot`` to crash:
```
katprep.clients.SessionExceptoin: 404: HTTP operation not successful {
  "error": {"message":"Resource user not found by id '1'"}
}
```

**Comment by stdevel** [comment]
Verified on RHEL7/Satellite 6.3 - investigating on branch `bug-empty_owner-issue82`.

**Comment by stdevel** [comment]
Fixed in commit f6ac11946e281ee15a191316011478c857a3a30f.
